src/types/api.types.ts

```diff
@@ -1221,9 +1219,112 @@ type TSocketEndpoints = {
         response: NewVirtualMoneyAccountResponse;
     };
     notifications_list: {
-        request: NotificationsListRequest;
-        response: NotificationsListResponse;
+        request: {
```

Comment:
for a note, this is temporary, backend already is aware that it needs to be added to BE definitions in order to appear in @deriv/api-types, so it can be removed later on,